### ISSUE TITLE
Expose overlay feature

### ISSUE THREAD

**Issue by nuest** [ropensci/qpdf#11]
`qpdf` has a nice feature to overlay one page over another. I'm looking for this package's equivalent of 

```r
system2("qpdf", args = c("--overlay", overlay_file_path, "--to=1", "--", base_file_path, output_file_path))
```

I'd like to contribute this issue, but have no experience with Rcpp, so I'm a bit lost, frankly.

**Comment by nuest** [comment]
@jeroen Would you be willing to point me in the right direction here? Since Heroku is discontinuing their free plans it would be great if I can use qpdf in my ShinyApp.

**Comment by jeroen** [comment]
@jberkenbilt can we do this with the C api?

**Comment by jeroen** [comment]
Ah here is an example https://github.com/qpdf/qpdf/blob/main/examples/pdf-overlay-page.cc

**Comment by jeroen** [comment]
@nuest can you give an example pdf file of what you need and what the output will look like?

**Comment by jeroen** [comment]
I've released this in qpdf 1.3.0 on cran. Lmk if this works for you.
